MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m

```diff
@@ -231,6 +233,12 @@ - (void) calculateBandwidthMetricFromAccessLog:(AVPlayerItemAccessLog *) log {
     }
 }
 
+- (void) updateViewingLivestream:(AVPlayerItemAccessLog *) log {
+    AVPlayerItemAccessLogEvent *lastEvent = log.events.lastObject;
+    NSString *playbackType = lastEvent.playbackType;
+    _playbackIsLivestream = [playbackType isEqualToString:@"LIVE"];
```

Comment:
oh nice, so this value is `VOD` if on demand?

Reply:
yes, or the key could be absent